packages/shared/providers.yaml

```diff
@@ -416,6 +416,7 @@ freshbooks:
         base_url: https://api.freshbooks.com
 freshdesk:
     auth_mode: BASIC
+    authorization_url: https://${connectionConfig.subdomain}.freshdesk.com/NOT/REAL # hack: so nango dashboard can request the connection config parameter
```

Comment:
This is not a suitable solution. There are other provider that also relies connectionConfig param that are only defined as part of `proxy_url`. We are gonna need to find a solution to support those while, not making param mandatory for provider like Salesforce